core/server/server.go

```diff
@@ -49,11 +51,19 @@ func NewCoreConfig(log logr.Logger, cfg *rest.Config, clusterName string) CoreSe
 }
 
 func NewCoreServer(cfg CoreServerConfig) pb.CoreServer {
+	ctx := context.Background()
+
 	cfgGetter := kube.NewImpersonatingConfigGetter(cfg.RestCfg, false)
+	k8s := kube.NewDefaultClientGetter(cfgGetter, cfg.clusterName)
+	c, _ := k8s.Client(ctx)
+	cacheContainer := cache.NewContainer(c, cfg.log)
```

Comment:
I am guessing this will need to go soon with #1675?

Reply:
Possibly yes.